scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -101,18 +101,27 @@ while true; do
             shift 2 ;;
          --binlogs)
             rc=$(echo $2 | sed -e 's/,/\n/g') ;
+            if [[ "$rc" != "true" &&  "$rc" != "false" ]]; then
+                usage; exit;
+            fi
             if [ "$rc" == "true" ]; then
                 CMD+=" --binaries"
             fi
             shift 2 ;;
```

Comment:
[suggestion]
if ( )
else if ( )
can be used here instead of two separate if.

Reply:
incorporated